### PR TITLE
1.18.2 with biome mappings, without biome generation

### PR DESCRIPTION
**Pull request by gniftygnome**
Alternative option in case it's useful:  I've applied the things Prospector did when accepting the Traverse 1.18.2 PR.

Comments:
I did not update Contributors (seems kind of maintainer-y plus maybe I don't know them all).
My biome mappings should be reviewed.  I might have picked the wrong targets and/or you might prefer other targets.
Terrestria mega tree saplings do not grow in either of my PRs.  I am not sure where to look but will poke at it more.

### CODE REVIEW

**Comment by gniftygnome** [comment]
I've tested this version converting from my other PR and also from an old 1.16.5 world.  In both cases, things worked as expected.  Minecraft gets kinda piggy when it's generating terrain at the edges of existing Terrestria chunks, but that's probably to be expected.  Upgrading from 1.16.5 seamlessly converted Redwood Clearing and Redwood Forest Edge to Redwood Forest.

File: src/main/java/com/terraformersmc/terrestria/mixin/MixinSchemas.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		Schema terrestria1_18_2 = builder.addSchema(2970, EMPTY_IDENTIFIER_NORMALIZE);
+		ImmutableMap<String, String> terrestria1_18_2_biome_map = ImmutableMap.<String, String>builder()
+				.put("terrestria:caldera", "minecraft:warm_ocean")
+				.put("terrestria:caldera_beach", "minecraft:beach")
+				.put("terrestria:caldera_foothills", "minecraft:plains")
+				.put("terrestria:caldera_ridge", "minecraft:meadow")
+				.put("terrestria:canyon_arches", "minecraft:desert")
+				.put("terrestria:canyon_cliffs", "minecraft:desert")
+				.put("terrestria:canyon_edge", "minecraft:desert")
+				.put("terrestria:dense_woodlands_edge", "minecraft_forest")
```
Comment (Prospector) [inline src/main/java/com/terraformersmc/terrestria/mixin/MixinSchemas.java]:
```suggestion
				.put("terrestria:dense_woodlands_edge", "minecraft:forest")
```

**Comment by gniftygnome** [comment]
> Terrestria mega tree saplings do not grow in either of my PRs. I am not sure where to look but will poke at it more.

OK they actually grow just fine, but I was testing on snowy grass and the snow must be cleared around the saplings (4x4 snow clear area, not 2x2).  This seems like a bug to me, but maybe not a new one?

Edit:  Turns out vanilla mega growth works the same way, so I think this is a non-issue.

**Comment by gniftygnome** [comment]
This PR is superseded by #269, which does everything this does and more and safely in combination with Traverse.

**Pull request closed without merging**